libraries/ol3/base.js

```diff
@@ -1497,7 +1497,11 @@ goog.inherits = function(childCtor, parentCtor) {
  * @return {*} The return value of the superclass method.
  */
 goog.base = function(me, opt_methodName, var_args) {
-  var caller = arguments.callee.caller;
+	function caller (){
+		return arguments.callee.caller;
+	}
+  var caller = caller();
```

Comment:
In the previous version arguments refer to the original function and with the change the caller will always be 'goog.base'?

Reply:
I'm not sure why we should have this in our libs. Maybe try removing it?